rero_ils/modules/circ_policies/api.py

```diff
@@ -59,6 +59,28 @@ class CircPolicy(IlsRecord):
     fetcher = circ_policy_id_fetcher
     provider = CircPolicyProvider
 
+    def extended_validation(self, **kwargs):
+        """Returns reasons for validation failures, otherwise True.
+
+        Override this function for classes that require extended validations
+        """
+        from ..patron_types.api import PatronType
+        from ..item_types.api import ItemType
```

Comment:
If possible then `imports` are better on top of file.

Reply:
Imports were done at the top of the file first, but it created circular references and needed to be moved into the function to work properly.